web/test/pages/onboarding/onboarding-shared.test.tsx

```diff
@@ -411,4 +413,25 @@ describe("onboarding - shared", () => {
       expect(screen.getByTestId("snackbar-alert-ERROR")).toBeInTheDocument();
     });
   });
+
+  describe("when query parameter sets onboarding flow to starting", () => {
+    beforeEach(() => {
+      jest.resetAllMocks();
+      useMockRouter({});
+      useMockUserData({});
+      useMockRoadmap({});
+      setupStatefulUserDataContext();
```

Comment:
I believe all of these functions are already being called at the top-level `beforeEach` in this file, so this `beforeEach` is redundant and could be removed